docs-android/app/src/main/res/values-pl/arrays.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is unnecessary, Android will use the default file in the "values" directory.